nextflow.config

```
@@ -236,43 +226,26 @@ manifest {
     homePage        = 'https://github.com/qbic-pipelines/vcftomaf'
     description     = """Convert vcf files to maf format"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=23.04.0'
+    nextflowVersion = '!>=24.04.2'
     version         = '1.0.0'
```

```suggestion
    version         = '1.1.0dev'
```

I had forgotten the back to dev PR, and just added it to #12 , so you might want to fix this here